numba/tests/test_lists.py

```diff
@@ -865,5 +865,270 @@ def test_reflect_clean(self):
         self.assertEqual([id(x) for x in l], ids)
 
 
+class TestListManagedElements(MemoryLeakMixin, TestCase):
+    "Test list containing objects that need refct"
+
+    def assert_list_element_precise_equal(self, expect, got):
+        self.assertEqual(len(expect), len(got))
+        for a, b in zip(expect, got):
+            self.assertPreciseEqual(a, b)
+
+    def test_reflect_passthru(self):
+        def pyfunc(con):
+            pass
+
+        cfunc = jit(nopython=True)(pyfunc)
```

Comment:
Could the repeated checking code for these tests be pulled out and called as a function?